autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -212,6 +237,10 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             id_columns : list, default = []
                 Banned subset of column names that model may not use as predictive features (e.g. contains label, user-ID, etc).
                 These columns are ignored during `fit()`, but DataFrame of just these columns with appended predictions may be produced, for example to submit in a ML competition.
+            set_best_to_refit_full : bool, default = False
+                If True, will set Trainer.best_model = Trainer.full_model_dict[Trainer.best_model]
+                This will change the default model used in Predictor for predictions when model is not specified to the refit_full version of the best model.
```

Comment:
This will change the default model used in Predictor for predictions when model is not specified to the refit_full version of the best model.
->
This will change the default model that Predictor uses for prediction when model is not specified to the refit_full version of the model that previously exhibit the highest validation score.

Reply:
Good language change, added